recipe/recipe.yaml

```diff
@@ -0,0 +1,81 @@
+context:
+  name: "networkx"
+  version: 3.4.2
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/n/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: 307c3669428c5362aab27c8a1260aa8f47c4e91d3891f48be0141738d8d053e1
+
+build:
+  number: 2
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=61.2
+    - pip
+  run:
+    - python >=3.10
+  run_constraints:
+    - numpy >=1.24
+    - scipy >=1.10,!=1.11.0,!=1.11.1
+    - matplotlib >=3.7
+    - pandas >=2.0
+
+tests:
+  - python:
+      imports:
+        - networkx
+        - networkx.algorithms
+        - networkx.algorithms.assortativity
+        - networkx.algorithms.bipartite
+        - networkx.algorithms.centrality
+        - networkx.algorithms.community
+        - networkx.algorithms.components
+        - networkx.algorithms.connectivity
+        - networkx.algorithms.coloring
+        - networkx.algorithms.flow
+        - networkx.algorithms.minors
+        - networkx.algorithms.traversal
+        - networkx.algorithms.isomorphism
+        - networkx.algorithms.shortest_paths
+        - networkx.algorithms.link_analysis
+        - networkx.algorithms.operators
+        - networkx.algorithms.approximation
+        - networkx.algorithms.tree
+        - networkx.classes
+        - networkx.generators
+        - networkx.drawing
+        - networkx.linalg
+        - networkx.readwrite
+        - networkx.readwrite.json_graph
+        - networkx.tests
+        - networkx.utils
```

Comment:
```suggestion
        - networkx.utils
  - requirements:
      run:
        - pip
    script:
      - pip check
```

Reply:
We don't need to do this anymore. `pip check` runs by default https://prefix-dev.github.io/rattler-build/latest/converting_from_conda_build/#converting-the-test-section